libbeat/filter/config.go

```diff
@@ -0,0 +1,14 @@
+package filter
+
+type DropFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type IncludeFieldsConfig struct {
+	Fields []string `config:"fields"`
+}
+
+type FilterConfig struct {
+	DropFields    *DropFieldsConfig    `config:"drop_fields"`
```

Comment:
For me `exclude_lines` is a way to skip some log lines when reading the log file (before even creating the event), while `drop_fields` is something that you would do after having the event as an "enhancement" of the event. I think it's important to have different naming "drop" vs "exclude" to make the difference between when the filtering is done: before creating the event vs after creating the event.
In addition I think `drop_fields` goes well with `drop_event`. If we would rename `drop_fields` to `exclude_fields` then we also have to rename `drop_event` with `exclude_event` which sounds a bit strange to me.